service/service.go

```diff
@@ -108,19 +153,22 @@ func New(config Config) (*Service, error) {
 
 	newService := &Service{
 		// Dependencies.
-		Deployer: deployerService,
-		Version:  versionService,
+		Deployer:   deployerService,
+		HelmClient: helmClient,
+		Version:    versionService,
 	}
 
 	return newService, nil
 }
 
-type Service struct {
-	// Dependencies.
-	Deployer deployer.Deployer
-	Version  *version.Service
-}
+func (s *Service) Boot(ctx context.Context) error {
+	// TODO Improve error handling https://github.com/giantswarm/giantswarm/issues/4965.
```

Comment:
JFYI https://github.com/giantswarm/fmt/blob/master/go/todo_comments.md

Reply:
Oh wow I did think it looked wrong! 😄 Fixed.